README.adoc

```diff
@@ -198,16 +208,142 @@ curl http://localhost:9080/hello
 ```
 endif::[]
 
-=== Tearing down the Docker container
+After you are finished checking out the application, stop your container by running the following command:
+
+[role='command']
+```
+docker stop springBootContainer
+```
+
+ifndef::cloud-hosted[]
+== Optional: Faster container startup with InstantOn
+
+Open Liberty InstantOn provides fast startup times for MicroProfile and Jakarta EE applications. In this section, you’ll learn how to build an InstantOn image that provides fast startup times. Your application can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features.
 
-To stop and remove your container, run the following commands:
+InstantOn requires a Linux system with kernel version 5.9 or greater. If your system does not meet this prerequisite, go to the <<running-the-application-on-open-liberty>> section. You can run the following command to check out the kernel version of your system:
+
+[role='command']
+```
+uname -r
+```
+
+The https://openliberty.io/docs/latest/reference/feature/crac.html[Coordinated Restore at Checkpoint^] (`crac`) feature enables applications to use the Liberty InstantOn implementation of the https://javadoc.io/doc/org.crac/crac/1.4.0/index.html[org.crac^] APIs. The Spring Framework hooks into the checkpoint and restore when the `org.crac` APIs are enabled.
+
+[role="code_command hotspot file=0", subs="quotes"]
+----
+#Replace the `Dockerfile` in the `start` directory.#
+`Dockerfile`
+----
+
+Dockerfile
+[source, Text, linenums, role='code_column']
+----
+include::instantOn/Dockerfile[]
+----
+
+crac.xml
+[source, xml, linenums, role='code_column']
+----
+include::finish/src/main/liberty/instantOn/crac.xml[]
+----
+
+To enable the `crac` feature, add the [hotspot=cracXml file=0]`COPY` command that copies the provided `crac.xml` configuration file to the Liberty `configDropins` configuration directory. The `crac.xml` file enables the [hotspot=crac file=1]`crac` feature to the Liberty instance.
```

Comment:
```suggestion
The [hotspot=cracXml file=0]`COPY` command copies the provided `crac.xml` configuration file to the Liberty `configDropins` configuration directory. The `crac.xml` file enables the [hotspot=crac file=1]`crac` feature to the Liberty instance.
```

Starting with "To enable" is unnecessary but also confusing because you've already got that line in the Dockerfile you've just replaced. It's not a step-by-step tutorial; the text is to explain the relevant bits of code in the files supplied.